modular_skyrat/modules/sec_haul/code/guns/token_system/token_system.dm

```diff
@@ -73,7 +73,7 @@
 	name = "sidearm armament holochip"
 	desc = "A holochip used in any armament vendor, this is for sidearms. Do not bend."
 	icon_state = "token_sidearm"
-	minimum_sec_level = SEC_LEVEL_BLUE
+	minimum_sec_level = SEC_LEVEL_AMBER
```

Comment:
PR desc says set to blue, yet you set to amber

Reply:
💀 I forgor